client/src/document/organisms/metadata/index.tsx

```diff
@@ -32,6 +37,11 @@ export function Metadata({ doc, locale }) {
           <LastModified value={doc.modified} locale={locale} />,{" "}
           <a href={`${doc.mdn_url}/contributors.txt`}>by MDN contributors</a>
         </p>
+        {locale.toLowerCase() === "en-us" &&
+          translations &&
+          !!translations.length && (
+            <LanguageMenu translations={translations} locale={locale} />
+          )}
```

Comment:
Now the LanguageMenu isn't appearing in non-en-US documents. 
Sad that the tests didn't notice this. 
I can help writing a headless test that views the French page and goes from there to English.

Reply:
So, this is by design remember. We said that on English pages you get the change language options which links to the language menu but on non-English, you get just the quick View to English link.

I believe since the work Florian did has landed it is now possible to also show the language menu on non-English pages but, perhaps we can make that change after we have landed this?